client/src/components/ProfileCard/ProfileCard.css

```diff
@@ -0,0 +1,37 @@
+.profileCard_card{
+    margin-top: 10px;
+    margin-bottom: 10px;
+    border: solid 2px red;
+    padding: 20px;
+    width: 100%;
+}
+.profileCard_content{
+    height: 300px;
+}
+.pic{
+    height: 100%;
+    width: 100%;
```

Comment:
I get what you're doing here, but normally I would see css files where they have either height or width set to something specific and the other will be set to auto. For example:
`height: 100%;
width: auto;`

This will allow you to modify one setting and have the other adjust accordingly.